Template10 (Library)/Controls/HamburgerMenu.xaml

```diff
@@ -168,9 +187,10 @@
                                             <Rectangle x:Name="CheckedBack"
                                                        Fill="{Binding NavButtonCheckedBackground, ElementName=ThisPage}"
                                                        IsHitTestVisible="False" Visibility="Collapsed" />
-                                            <ContentControl x:Name="NormalFore" AutomationProperties.AccessibilityView="Raw"
```

Comment:
Good, but please ensure the ToggleButton does not appear in the control list of the narrator now that you have removed this raw value.